py-polars/polars/config.py

```diff
@@ -416,6 +443,27 @@ def set_fmt_float(cls, fmt: FloatFmt | None = "mixed") -> type[Config]:
         fmt : {"mixed", "full"}
             How to format floating point numbers.
 
+        Examples
```

Comment:
This example does not help me understand what "mixed" float formatting means.

Reply:
Done, I've added some more text and a better example with large/small numbers